RadixWallet/Features/DappInteractionFeature/Coordinator/DappInteractionFlow.swift

```diff
@@ -246,7 +248,7 @@ struct DappInteractionFlow: Sendable, FeatureReducer {
 			state.responseItems[.remote(.auth(.usePersona(item)))] = .remote(.auth(.usePersona(.init(
 				persona: .init(
 					identityAddress: persona.address,
-					label: persona.displayName
+					label: persona.displayName.rawValue
```

Comment:
Which one should we use, `displayName.rawValue` or `displayName.value`?

Reply:
we can use move over to `value`. I've tried to make minimal changes... basically we wanna land this PR ASAP, and it shuold not make any changes to how the wallet app works. 

Then later we can update, improve and fix the code in iOS wallet and iterate